themes-default/slim/src/components/subtitle-search.vue

```diff
@@ -0,0 +1,258 @@
+                        <span class="subtitle-name">{{props.row.filename}}</span>
+                        <img v-if="props.row.sub_score >= props.row.min_score" src="images/save.png" width="16" height="16"/>
+                    </a>
+                </span>
+                <span v-else-if="props.column.field == 'download'">
+                    <a :title="`Download ${props.row.hearing_impaired ? 'hearing impaired ' : ' '} subtitle: ${props.row.filename}`" @click="pickSubtitle(props.row.id)">
+                        <img src="images/download.png" width="16" height="16"/>
+                    </a>
+                </span>
+                <span v-else>
+                    {{props.formattedRow[props.column.field]}}
+                </span>
+            </template>
+        </vue-good-table>
+    </td>
+</tr>
+</template>
+<script>
+
+import { mapState } from 'vuex';
+import { VueGoodTable } from 'vue-good-table';
+
+export default {
+    name: 'subtitle-search',
+    components: {
+        VueGoodTable
+    },
+    props: {
+        show: {
+            type: Object,
+            required: true
+        },
+        season: {
+            type: [String, Number],
+            required: true
+        },
+        episode: {
+            type: [String, Number],
+            required: true
+        }
+    },
+    data() {
+        return {
+            columns: [{
+                label: 'Filename',
+                field: 'filename'
+            }, {
+                label: 'Language',
+                field: 'lang'
+            }, {
+                label: 'Provider',
+                field: 'provider'
+            }, {
+                label: 'Score',
+                field: 'score',
+                type: 'number'
+            }, {
+                label: 'Sub Score',
+                field: 'sub_score',
+                type: 'number'
+            }, {
+                label: 'Missing Matches',
+                field: rowObj => {
+                    if (rowObj.missing_guess) {
+                        return rowObj.missing_guess.join(', ');
+                    }
+                },
+                type: 'array'
+            }, {
+                label: 'Download',
+                field: 'download'
+            }],
+            subtitles: [],
+            displayQuestion: false,
+            loading: false,
+            loadingMessage: ''
+        };
+    },
+    computed: {
+        ...mapState({
+            config: state => state.config
+        })
+    },
+    mounted() {
+        this.displayQuestion = true;
+    },
+    destroyed() {
+        // Remove the element from the DOM
+        this.$el.parentNode.removeChild(this.$el);
+    },
+    methods: {
+        autoSearch() {
+            const { episode, season, show } = this;
+
+            this.displayQuestion = false;
+            const url = `home/searchEpisodeSubtitles?indexername=${show.indexer}&seriesid=${show.id[show.indexer]}&season=${season}&episode=${episode}`;
```

Comment:
Let `Axios` handle the heavy lifting.
```suggestion
            const params = {
                indexername: show.indexer,
                seriesid: show.id[show.indexer],
                season,
                episode
            };
```
Also, I suggest that instead of doing that *every time* you need the `params`,
just make a computed property that returns this object, and use it in all three locations.